lib/kovid/cli.rb

```diff
@@ -56,7 +56,16 @@ def compare(*name)
 
     desc 'states STATE STATE', 'Returns full comparison table for the given states. Accepts multiple states.'
     def states(*states)
-      puts Kovid.states(states)
+      # This ensures this command is case insensitive.
+      downcased_states = states.map(&:downcase)
+
+      puts Kovid.states(downcased_states)
+      data_source
+    end
+
+    desc 'all_us_states', 'Returns full comparison table for all US states'
+    def all_us_states
```

Comment:
Can you please make an alias for this so it works with say `kovid aus`? To type `kovid all_us_states` every time seems tedious, a shortcut would be nice :)